src/runtime_config/default_actions.js

```diff
@@ -68,6 +68,34 @@ function checkLength(value, min, max, field) {
   }
 }
 
+function checkNone(args) {
+  if (args !== null && args !== undefined) {
+    throw new Error(`Proposal does not accept any argument, found \"${args}\"`);
+  }
+}
+
+function getUniqueKvKey() {
```

Comment:
This sounds more like random uuid. I don't have a good alternative, maybe `getSingletonKvKey()`?

Reply:
`getWellKnownKvKey